packages/google-closure-compiler/lib/grunt/index.js

```diff
@@ -218,6 +224,31 @@ module.exports = (grunt, pluginOptions) => {
       });
   }
 
+  // processPromises function grabs `ps` as array of promise-returning functions and `done` function as callback. It separates `ps` into batches of length == compileInBatches and runs resulting batches in parallel.
```

Comment:
The description looks good, but can you convert this to a JSDoc comment block and specify the types of the parameters as well as the return value? Here's an example https://github.com/google/closure-compiler-npm/blob/master/packages/google-closure-compiler/lib/gulp/index.js#L48-L51

Also - watch the line length. Try to keep it 100 characters or shorter.

Reply:
Ok, will do this tomorrow.

Reply:
Done